mobile/src/core/translations.ts

```diff
@@ -1,9 +1,27 @@
-export const core = {
+export const core: any = {
   actions: {
     cancel: 'core.actions.cancel',
     ok: 'core.actions.ok',
     submit: 'core.actions.submit'
   },
+  dates: {
+    months: {
+      short: {
+        0: 'core.dates.months.short.0',
```

Comment:
when talking about months, it's probably less confusing to start at 1 instead of zero.

Reply:
JS start count months from 0, but every other parts of the date from 1. So it's just a cpatibility thing